cob_camera_sensors/package.xml

```diff
@@ -8,28 +8,22 @@
   <url type="website">http://ros.org/wiki/cob_camera_sensors</url>
   <!-- <url type="bugtracker"></url> -->
 
-  <maintainer email="jsf@ipa.fhg.de">Jan Fischer</maintainer> 
+  <maintainer email="jsf@ipa.fhg.de">Jan Fischer</maintainer>
   <author email="jsf@ipa.fhg.de">Jan Fischer</author>
 
   <buildtool_depend>catkin</buildtool_depend>
 
-  <build_depend>cob_vision_utils</build_depend>
-  <build_depend>cv_bridge</build_depend>
-  <build_depend>image_transport</build_depend>
-  <build_depend>message_filters</build_depend>
   <build_depend>message_generation</build_depend>
-  <build_depend>polled_camera</build_depend>
-  <build_depend>roscpp</build_depend>
-  <build_depend>sensor_msgs</build_depend>
-  
-  <run_depend>cob_vision_utils</run_depend>
-  <run_depend>cv_bridge</run_depend>
-  <run_depend>image_transport</run_depend>
-  <run_depend>message_filters</run_depend>
-  <run_depend>message_runtime</run_depend>
-  <run_depend>polled_camera</run_depend>
-  <run_depend>roscpp</run_depend>
-  <run_depend>rospy</run_depend>
-  <run_depend>sensor_msgs</run_depend>
+  <exec_depend>message_runtime</exec_depend>
+
+  <depend>cob_vision_utils</depend>
+  <depend>cv_bridge</depend>
```

Comment:
The dependency is `libopencv-dev`...adding it for now.
However, I'd like to clean up `cob_camera_sensors` anyway as these .cpp files are nowhere used (no executable, no library)!

@ipa-rmb:
Can you drop a line here, where and which files from `cob_camera_sensors` and `cob_vision_utils` are used? Maybe we can move these packages into `cob_perception_common`?